### PR TITLE
Add Russian translations of documents

### PR DESCRIPTION
**Pull request by sadykhzadeh**
## Description
We have translated all .md files into Russian

### CODE REVIEW

**Comment by ouuan** [comment]
> No, "Artifacts" can be translated. For example https://ru.werf.io/documentation/configuration/stapel_artifact.html

Do you think users can find where to download if they see the translation but never heard of Artifacts of GitHub Actions? If so, you can translate it.

**Comment by IZOBRETATEL777** [comment]
> > No, "Artifacts" can be translated. For example https://ru.werf.io/documentation/configuration/stapel_artifact.html
> 
> Do you think users can find where to download if they see the translation but never heard of Artifacts of GitHub Actions? If so, you can translate it.

As English-speaking people can diff artifacts of GitHub and artifacts of Ancient Egypt as Russian-speaking people can do it.
For best identification I didn't translate subtitle.

**Comment by ouuan** [comment]
I mean, on GitHub, it's displayed as "Artifacts" in English:

image

Do you think they can find this by the translation?

**Comment by IZOBRETATEL777** [comment]
> I mean, on GitHub, it's displayed as "Artifacts" in English:
> 
> image
> 
> Do you think they can easily find this by the translation?

Why not? In this context it looks the same and sounds the same (in Russian [artefakty]). I think people can identify this.

**Comment by ouuan** [comment]
> Why not? In this context it looks the same and sounds the same (in Russian [artefakty]). I think people can identify this.

I wasn't asking a rhetorical question. I don't know because I don't speak Russian. I've said, "If so, you can translate it". However, you can choose whether to translate or not, because I don't speak Russian.

File: CONTRIBUTING_ru-RU.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+**Впервые в Qt?** Не бойтесь, Qt не трудно изучить. Сначала вы должны установить Qt >= 5.12 и CMake >= 3.5. Пользователи Linux могут установить их с помощью диспетчера пакетов в вашей системе. Пользователи Windows могут скачать Qt [здесь](https://www.qt.io/offline-installers), также рекомендовано использовать [Visual Studio](https://visualstudio.microsoft.com/) и скачать плагин CMake. Прочитайте [сборку из исходного кода](doc/INSTALL.md#Build-from-source), чтобы узнать, как делать build. Прочтите [документы Qt](https://doc.qt.io/) о том, как использовать встроенные виджеты Qt, как проектировать пользовательский интерфейс и как использовать [сигналы и слоты](https://doc.qt.io/qt-5/signalsandslots.html). Обычно Вам не нужно использовать стандартное пространство имен std. Используйте `QString` вместо `std::string`.
```
Comment (ouuan) [inline CONTRIBUTING_ru-RU.md:20]:
```suggestion
**Впервые в Qt?** Не бойтесь, Qt не трудно изучить. Сначала вы должны установить Qt >= 5.12 и CMake >= 3.5. Пользователи Linux могут установить их с помощью диспетчера пакетов в вашей системе. Пользователи Windows могут скачать Qt [здесь](https://www.qt.io/offline-installers), также рекомендовано использовать [Visual Studio](https://visualstudio.microsoft.com/) и скачать плагин CMake. Прочитайте [сборку из исходного кода](doc/INSTALL_ru-RU.md#Сборка-из-исходного-кода), чтобы узнать, как делать build. Прочтите [документы Qt](https://doc.qt.io/) о том, как использовать встроенные виджеты Qt, как проектировать пользовательский интерфейс и как использовать [сигналы и слоты](https://doc.qt.io/qt-5/signalsandslots.html). Обычно Вам не нужно использовать стандартное пространство имен std. Используйте `QString` вместо `std::string`.
```

**Comment by ouuan** [comment]
Did you check the "allow edits from maintainers" option when creating this PR?

**Comment by sadykhzadeh** [comment]
> Did you check the "allow edits from maintainers" option when creating this PR?

No

**Comment by ouuan** [comment]
Have you finished?

**Comment by IZOBRETATEL777** [comment]
> Have you finished?

Over 100% :+1:

**Comment by ouuan** [comment]
@sadykhzadeh @IZOBRETATEL777 I just realized we may need a warning, which says "This document is translated from the English version, it's not guaranteed to be up to date". You can open another PR for this, or tell me the proper translation.

**Comment by ouuan** [comment]
[INSTALL.md](https://github.com/cpeditor/cpeditor/blob/master/doc/INSTALL.md) has been updated in 7a087f1, you can translate it.

**Pull request merged**